common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -1,21 +1,41 @@
-"""Implements basics of Capa, including class CapaModule."""
-
+"""
```

Comment:
I'd love to suggest that we call this `capa_block.py` and move its location. But I won't! There seems to be quite a bit of other renaming to do before that happens. This PR seems like a solid first step towards that eventual future.

Reply:
Agreed--I was tempted to do that in this PR too, but kept myself to the less-risky changes :smile: I think @symbolist and team will likely get to do some renaming and cleanup of these modules in the next phase of BD-04.